### DISCUSSION TITLE
How to completely revert to stock?

### DISCUSSION THREAD

**Discussion by vladniko75** [linearmouse/linearmouse#567] [Q&A]
I am new to Mac OS, was messing about with linear mouse and the acceleration feature. Anyways I couldn't get a config I liked, so I reset it to stock and uninstalled the app. I also set the trackpad settings in iOS to stock. Still something changed recently when I was messing with linear mouse, and right before I uninstalled it it should be stock but it feels very strange. Anyway to refresh stock everything for trackpad firmware?

2023 MBP 14" M2 Pro

**Comment by lujjjh** [comment]
Can you please provide more specific details? Specifically, what exactly do you find strange about the trackpad? LinearMouse will not modify any persistent settings in the system. When you exit LinearMouse, all settings will be restored. If the trackpad still behaves strangely after uninstalling LinearMouse and restarting your Mac, you should check if you have made any modifications in the System Settings.

**Comment by vladniko75** [comment]
I Will check again but everything is stock. Its truly indescribable, its as if its laggy but it isn't, its more so it feels like it doesn't go exactly where you want it to. out of the box, it felt great, and up to this point it did, but as soon as I messed with that one setting the cursor isn't going exactly where my finger wants it as before. there no backups online I can restore or lines of code I can run in terminal to make sure its all set to default?

**Reply by lujjjh** [reply]
I'm not quite sure if I fully understand your question. LinearMouse stores its configuration in `~/.config/linearmouse/linearmouse.json`. If you want to reset the configuration of LinearMouse, you can delete this file and reload the configuration in LinearMouse. Also, you can manually backup and restore the configuration. However, once LinearMouse has been exited, your trackpad settings should always revert to the default system settings, no matter how much you may have altered the settings within the program.

**Comment by vladniko75** [comment]
I deleted Linearmouse as a whole, but I still feel like something system
side is different than how it was when I took it out of the box sunday.

**Reply by lujjjh** [reply]
Please reply in the answer instead of creating ones.

Like I mentioned earlier, if you uninstall LinearMouse, restart your computer, and the problem continues to exist, it's probably not caused by LinearMouse.

I would suggest you create a new macOS user and log into it to see if the problem still persists.

**Reply by vladniko75** [reply]
apologies I was just replying in email. I ended up getting a new mac and not touching the mouse settings. new mac Was needed because the 5 day old one I got had chips in screen
